analysis2_manuscript.tex

```diff
@@ -351,7 +351,7 @@ \section*{Implementing Selection in \stdpopsim}
     coefficients are drawn from a specified DFE) occur within annotation regions (e.g. exons), while all mutations
     occurring outside of these regions do not affect fitness.
     There are relatively few species with published DFE estimates in the literature.
-    \stdpopsim currently has implemented DFEs for five species in the catalogue:
+    \stdpopsim currently has implemented DFEs for five species in the catalog:
```

Comment:
US spelling seemed more prevalent throughout, so I changed over a couple of UK spellings.